test/e2e/framework/util.go

```diff
@@ -90,3 +90,28 @@ func NewUUID() string {
 	lastUUID = result
 	return result.String()
 }
+
+func ClearAllImages(client *FraktiClient) {
+	imageList, err := client.ListImages(nil)
+	ExpectNoError(err, "Failed to get image list: %v", err)
+	for _, image := range imageList {
+		if len(image.RepoTags) == 0 {
+			for _, rd := range image.RepoDigests {
+				err = client.RemoveImage(&kubeapi.ImageSpec{
+					Image: &rd,
```

Comment:
No,  it is not recommended to take the address of range variables because Golang reused it for whole loop.